rancher2/resource_rancher2_user_test.go

```diff
@@ -26,6 +26,7 @@ resource "` + testAccRancher2UserType + `" "foo" {
   username = "foo"
   password = "TestACC123456"
   enabled = true
+  must_change_password = true
 }
```

Comment:
It looks like there should be a test where `must_change_password` is `false`?

Reply:
test added :)